ArduSub/ArduSub.cpp

```diff
@@ -479,4 +486,14 @@ bool Sub::ensure_ekf_origin()
     return true;
 }
 
+#if RC_ENABLED
```

Comment:
Need to do a lot better on this name

`AP_SUB_RC_ENABLED`

Reply:
done